dietpi/dietpi-software

```diff
@@ -12819,6 +12819,7 @@ If no WireGuard (auto)start is included, but you require it, please do the follo
 			Remove_Service adsb-docker
 			Remove_Service adsb-feeder-update
 			Remove_Service adsb-setup
+			[[ -f /opt/adsb/pre-uninstall-cleanup ]] && G_EXEC /opt/adsb/pre-uninstall-cleanup
```

Comment:
right now this script doesn't exist... but this seems like the easy way to make sure that once I have figured out a clean and consistent and easy to maintain way to do this for all images that the ADS-B Feeder might install now (and that can easily be kept current by upstream without having to make changes in DietPi)

Reply:
Sounds reasonable.